docs/java/README.md

```diff
@@ -37,3 +37,4 @@ javaApplication {
 
 - [Declarative Gradle for Kotlin](../kotlin/README.md)
 - [Declarative Gradle for Android](../android/README.md)
+- [Declarative Gradle for Swift](../swift/README.md)
```

Comment:
The idea here was to rather reference the related pages and not all of them (e.g. Kotlin and Android stem from Java), but we can link all of them for now until we have more complex navigation